src/utils/index.tsx

```diff
@@ -66,9 +66,8 @@ export const timeLeft = (end): string => {
 
   if (yearsDiff > 0) return yearsDiff === 1 ? `${yearsDiff} year` : `${yearsDiff} year`
   if (monthsDiff > 0) return monthsDiff === 1 ? `${monthsDiff} month` : `${monthsDiff} months`
-  if (weeksDiff > 0) return weeksDiff === 1 ? `${daysDiff} week` : `${daysDiff} week`
```

Comment:
was the `week` label removed for some reason?

Reply:
At first, I thought about fixing this by using the right variable (`weekDiff`) but the result would have been "1 week" instead of "13 days" and I think this is incorrect as well.